src/DataFactory/CompositeDataFactory.js

```diff
@@ -17,6 +18,7 @@ class CompositeDataFactory {
         this._factories = [
             new PrimitiveDataFactory(this),
             new ListDataFactory(this),
+            new SetDataFactory(this),
```

Comment:
It doesn't works correctly if I don't have Set.aes included or have a custom `Set` implementation, like:
```
contract interface Set =
  type set = int
```
🙃 
Can the ACI of "Set.aes" be implicitly added if "Set.aes" is included in contract ACIs provided by user? Or ACIs doesn't explain imports?

Reply:
Interesting edge case, but there is no include information in the ACI and I don't see any other way to workaround this.